SingularityService/src/main/java/com/hubspot/singularity/smtp/SingularitySmtpSender.java

```diff
@@ -62,6 +68,10 @@ public void stop() {
   }
 
   public void queueMail(final List<String> toList, final List<String> ccList, final String subject, final String body) {
+    if (LOG_EMAILS_FOR_DEBUG) {
+      LOG.trace("TheMail: " + body);
```

Comment:
For readability I've found it to be easier to put the recipients first and the body last so you're not scrolling through a wall of HTML looking for the recipient at the end